drivers/gles3/storage/texture_storage.h

```diff
@@ -364,6 +375,10 @@ class TextureStorage : public RendererTextureStorage {
 
 	RID_Owner<CanvasTexture, true> canvas_texture_owner;
 
+	/* CANVAS SHADOW */
+
+	RID_PtrOwner<CanvasLightShadow> canvas_light_shadow_owner;
```

Comment:
I'm not sure if texture_storage is the right place for shadow textures, like this. But I haven't implemented 2D lights yet, so its fine to place here and when I get to 2D lights I can move it if necessary.

Reply:
I wasn't sure either, had to place it somewhere, open to suggestions or we can just leave it there for now seeing this so far seems to impact GLES3 only

Reply:
Yeah, I'm not even sure I will be using a CanvasLightShadow struct like that. Let's just leave it there for now.